fastcrawler/utils/injection.py

```diff
@@ -47,36 +47,35 @@ def __repr__(self) -> str:
 
     @staticmethod
     async def inject(func: "_Depends"):
+        sig = inspect.signature(func.dependency)
+        bound = sig.bind_partial(*func.args, **func.kwargs)
+        bound.apply_defaults()
+
+        for name, value in bound.arguments.items():
+            if isinstance(value, _Depends):
+                bound.arguments[name] = await _Depends.evaluate_dependency(value)
+
```

Comment:
why did you change the codes here? any reason?

Reply:
To reduce mypy errors and write in better way (it seems has been duplicate code in the algorithm)
This part of code not related to this pr.
we can revert this part